src/app/[locale]/projects/edit/[id]/components/EditProject.tsx

```diff
@@ -226,9 +233,9 @@ function EditProject({ projectId }: { projectId: string }): JSX.Element {
                                         project["_embedded"]["leadArchitect"].fullName ?? '' })
                 }
     
-                if (project["_embedded"]?.["projectOwner"] !== undefined) {
-                    setProjectOwner({ [project["_embedded"]["projectOwner"].email]:
-                                       project["_embedded"]["projectOwner"].fullName ?? '' })
+                if (project["projectOwner"] !== undefined) {
+                    setProjectOwner({ 'Project owner':
```

Comment:
project owner must have format
`{email : full name}`
for example:
{ 'admin@sw360.org': 'Test Admin'}